### PR TITLE
fixes #4691 - add search for hosts' managed status

### PR DESCRIPTION
**Pull request by mmoll**
a DB migration is needed, as this field has no default value at the moment

### CODE REVIEW

**Comment by domcleal** [comment]
[test] random failure, you're ok

File: db/migrate/20140318221513_change_host_managed_default_to_false.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class ChangeHostManagedDefaultToFalse < ActiveRecord::Migration
+  def self.up
+    Host.update_all({:managed => false}, {:managed => nil})
```
Comment (domcleal) [inline db/migrate/20140318221513_change_host_managed_default_to_false.rb]: Nitpick: `Host.unscoped.update_all`

**Comment by domcleal** [comment]
I think this is sane, especially as you've changed the default value.  @GregSutcliffe any reason you can think of why this wouldn't be ok?  Even in unattended=false, managed should always be false too.

File: db/migrate/20140318221513_change_host_managed_default_to_false.rb (hunk 1 above)
Comment (mmoll) [inline db/migrate/20140318221513_change_host_managed_default_to_false.rb]: fixed :)

**Comment by GregSutcliffe** [comment]
@domcleal @mmoll looks good to me

File: db/migrate/20140318221513_change_host_managed_default_to_false.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class ChangeHostManagedDefaultToFalse < ActiveRecord::Migration
+  def self.up
+    Host.unscoped.update_all({:managed => false}, {:managed => nil})
```
Comment (dmitri-d) [inline db/migrate/20140318221513_change_host_managed_default_to_false.rb]: I'd rewrite this as: Host.unscoped.where(:managed => nil).update_all(:managed => false).
ActiveRecord > 4 doesn't support conditions in update_all.

Comment (mmoll) [inline db/migrate/20140318221513_change_host_managed_default_to_false.rb]: Good to know, changed that. :bow:

**Comment by domcleal** [comment]
Merged as 07a1bb3 for Foreman 1.5.0, thanks @mmoll!

**Pull request closed without merging**